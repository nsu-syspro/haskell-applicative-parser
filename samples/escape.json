{"data": [
  {
    "key": 0,
    "value": "test"
  },
  {
    "key": 1,
    "value": "something \"quoted\""
  },
  {
    "key": 2,
    "value": "one\\two\\\"three\""
  }
]}
