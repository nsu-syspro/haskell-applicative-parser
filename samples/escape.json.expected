{"data": [{"key": 0.0, "value": "test"}, {"key": 1.0, "value": "something \"quoted\""}, {"key": 2.0, "value": "one\\two\\\"three\""}]}